addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_category.js

```diff
@@ -29,7 +28,7 @@ patch(SlideUploadCategory.prototype, {
             ? this.state.choices.certifications.find(
                   (c) => c.value === this.state.choices.certificationId
               ).label
-            : _t("Select or create a certification");
+              : _t("Select a certification");
```

Comment:
Can we keep the same format as before, with the ":" aligned vertically to the "?" ?